recipe/scripts/activate.bat

```diff
@@ -1,2 +1,5 @@
 set "JAVA_HOME_CONDA_BACKUP=%JAVA_HOME%"
 set "JAVA_HOME=%CONDA_PREFIX%\Library"
+
+set "JAVA_LD_LIBRARY_PATH_BACKUP=%JAVA_LD_LIBRARY_PATH%"
+set "JAVA_LD_LIBRARY_PATH=%JAVA_HOME%\jre\bin\server"
```

Comment:
Was hoping this would help the issue on Windows (as it does on Unix), but it seems to be ignored AFAICT. If requested, we can drop it, but would slightly prefer to keep (though don't have strong feelings on it).

Reply:
As far as I know there is no `LD_LIBRARY_PATH` on Windows, so what would be the purpose of adding a `JAVA_LD_LIBRARY_PATH`?

Reply:
True. Wasn't sure if Java still defined it's own `JAVA_LD_LIBRARY_PATH` for Windows anyways. Can drop it if that is not the case.